build/charts/antrea/templates/crds/networkpolicy.yaml

```diff
@@ -152,6 +153,16 @@ spec:
                                   type: integer
                                   minimum: 0
                                   maximum: 255
+                            igmp:
```

Comment:
I agree ACNP is more important. But I like to understand what is the issue you saw to support ANP? Why members in different Namespaces is an issue?

Reply:
In my opinion, there should be no issue on the datapath implementation for egress ANP rules with multicast traffic. But for IGMP rules, it is a bit confused on the meaning of an ANP rule. For example, we have an ANP rule to only allow IGMP report from ns1/Pod1, then we could drop reports from Pods in ns1, but what action shall we take on the Pod reports from a different namespace ( ns2 )? Of cource, we could have another ANP rule to define the behaviors in ns2, but it is a bit wired to add ANP rules per namespace on the IGMP report to the same group, because the multicast is actually to forward the packet to many destinations without differences.

Reply:
We use Pod + Namespace selector to select IGMP source Pods right? So, Pods from any Namespace can be selected. Why you say we need another rule for ns2?

Reply:
I think what wenying meant is if we want to drop IGMP Query to pods which matches label selector such as `app=mypod` from ns1 and ns2. For acnp, one rule with podselector is enough. But if we use anp, we have to define two ANPs with same rule for ns1 and ns2.

Reply:
ANP rule that drops IGMQ query applies to Pods in this namespace only, right? For namespace users, they can only control ingress and egress of their own Pods. The purpose of having ACNP and ANP is to separate permissions, and grant namespace users the permissions of their own Pods.

Reply:
OK, then let's keep the ANP change.